bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -117,30 +122,99 @@ public void componentShown(final ComponentEvent componentEvent) {
             updateDetailEditorEnabledState(enabled);
         }
 
-        final TargetEnvironment env = getDataObject().getTargetEnvironment();
-        _w.tag.setEnabled(enabled && env.getBase() != _curPos);
-
         final SPInstObsComp inst = getContextInstrumentDataObject();
         _w.newMenu.setEnabled(enabled && inst != null);
+        _w.tag.setEnabled(enabled && selectionIsBasePosition());
```

Comment:
Shouldn't it be `!selectionIsBasePosition()`.  It seems like we've reversed the logic here?